mandatory/2-function-creation.js

```diff
@@ -14,27 +18,62 @@ function isAcceptableUser(userAge, isLoggedIn) {}
   The function should return the new price to be paid (e.g. if the totalPrice is 150, a 5% discount
     is applieds and 142.5 should be returned)
 */
+var discount;
+function applyDiscount(totalPrice) {
+  if(totalPrice>200){
+    discount=10;}
+    else{ discount=5;
+
+    }
+    return  totalPrice-( totalPrice * discount / 100);
+  }
 
-function applyDiscount(totalPrice) {}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
   */
-function printOddNumbers(limit) {}
+function printOddNumbers(limit) {
+  let i = 1;
+  
+  while (i % 1===0){
+    console.log(i);
+  }
+  i = i + 1;
+}
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
-function buyTwoGetTheCheapestFree(price1, price2) {}
-
+function buyTwoGetTheCheapestFree(price1, price2) {
+  let totalPrice = price1 + price2;
+  if (price1 < price2){
+    let discount = price1;
+  }
+  return  totalPrice - price1;
+  }
```

Comment:
When you ran your unit tests did they pass for you?

consider following 

`buyTwoGetCheapestFree(200,100)` would your code still work correctly?